packages/ember-data/lib/adapters/build-url-mixin.js

```diff
@@ -27,7 +27,7 @@ var get = Ember.get;
   @class BuildURLMixin
   @namespace DS
 */
-var BuildURLMixin = Ember.Mixin.create({
+export default Ember.Mixin.create({
```

Comment:
Looks like this line is the only chase where @stefanpenner changed a file to use `export default` without a name. I think we should merge this pr as is to unblock users who want to test Ember 2.0.

We can bike shed this topic as a separate issue since the Ember Data codebase already contains a mix of both styles.